docs/how_to/use_optimizationlevel.rst

```diff
@@ -0,0 +1,275 @@
+#####################################
+Setting transpiler optimization level
+#####################################
+
+This guide describes the effects of the ``optimization_level``
+parameter of :func:`~qiskit.compiler.transpile`.
+
+``optimization_level`` helps you to optimize your quantum circuit.
+This parameter takes an integer which can be a value between 0 and 3,
+where the higher the number, the more optimized the result.
+You can find more information about this parameter's value and its meaning for
+the internals of the :mod:`~.transpiler` in: :ref:`working_with_preset_pass_managers`.
+
+Initialize the quantum circuit
+==============================
+
+For this example, you will explore the `CSWAP <https://qiskit.org/documentation/stubs/qiskit.circuit.QuantumCircuit.cswap.html>`_ gate,
```

Comment:
I assume you did a hyperlink to the documentation so you could use `CSWAP` instead of `CSwapGate`? If so it might be better to do:

```suggestion
For this example, you will explore the :ref:`CSWAP <:class:~.CSwapGate>` gate,
```
(I think that's the correct syntax).